bids2openminds/mapping.py

```diff
@@ -0,0 +1,114 @@
+experimental_approaches={

```

Comment:
Another suggestion: use the convention to constances in python and add map to distinguish between openminds property names in python module.

Suggestion: MAP_2_EXPERIMENTAL_APPROACHES or MAP_TO_EXPERIMENTAL_APPROACHES